index.html

```diff
@@ -22,5 +22,29 @@
         </div>
       </nav>
     </header>
+    <main class="main">
+      <h1 class="h1">I'M Charle Peter. Glad to see you!</h1>
+      <p class="p">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h4 class="h4-connect">LET'S CONNECT</h4>
+      <div class="flex item-center">
+        <a class="icon-tag" href="#"
+          ><img src="/image/twitter.png" alt="twitter"
+        /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/linkedin.png" alt="linkedin"
+        /></a>
+        <a class="icon-tag" href="#"><img src="/image/m.png" alt="m" /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/git.png" alt="Github"
+        /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/angelist.png" alt="angelist"
+        /></a>
+      </div>
```

Comment:
**_kindly fix those styling issues to match the Figma design_** :

- [ ] kindly increase the margin (spaces) between the section title & the navbar
- [ ] kindly decrease the width of the title to match the design, I would suggest adding `max-width:75%;` to the `<h1>` tag
- [ ] kindly reduce the font weight for the `<p>` tag & the `Let's Connect` title to match Figma, currently, they are much bolder than it supposes to be
- [ ] kindly add the color `#F7F7F9` to the body's style and add the missing border-radius in the bottom left corner of the deadline section
```CSS
 body : {background: #F7F7F9;}
```

Your Design           | Figma Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/191862990-d0cefaa8-0986-4f76-8489-3c22cd812b8f.png)| ![image](https://user-images.githubusercontent.com/89282221/191863261-20ed723b-c2b1-44bc-97ab-af203bd27f2b.png)